sparkles/tests/test_review.py

```diff
@@ -182,3 +184,113 @@ def test_run_aca_review_function():
         {'text': 'P2: 2.84 less than 3.0 for ER', 'category': 'critical'},
         {'text': 'ER count of 9th (8.9 for -9.9C) mag guide stars 1.91 < 3.0',
          'category': 'critical'}]
+
+
+def test_roll_outside_range():
+    kw = {'att': [-0.82389459, -0.1248412, 0.35722113, 0.42190692],
```

Comment:
OK, just needs a docstring with reference to the PR #91 this is testing and an explanation of why this had previously failed.  (I.e just say in a sentence that this attitude ends up with roll outside of the internally-computed roll_min / roll_max.).